### ISSUE TITLE
Converted Model from Diffusers Dreambooth -> to CKPT dont work in stable-diffusion-webui ?

### ISSUE THREAD

**Issue by janoschsimon** [huggingface/diffusers#1730]
Hey there first thex for the amazing work :) i manged to get Dreambooth runnding on my trust 2080ti and the training works i have results in the 400 and 800 folder, but when i try to load them they dont apear in stable-diffusion-webui. im using his script for the conversion to ckpt https://gist.github.com/Christopher-Hayes/636ba25e0ae2e7020722d5386ac2571b are there any other convertes i might try? 
here is the code i run dreambooth with. i can also upload the ckpt to gdrive if that helps?

export MODEL_NAME="runwayml/stable-diffusion-v1-5"
export INSTANCE_DIR="training"
export CLASS_DIR="classes"
export OUTPUT_DIR="../../../output"

accelerate launch train_dreambooth.py \
  --pretrained_model_name_or_path=$MODEL_NAME \
  --pretrained_vae_name_or_path="stabilityai/sd-vae-ft-mse" \
  --instance_data_dir=$INSTANCE_DIR \
  --class_data_dir=$CLASS_DIR \
  --output_dir=$OUTPUT_DIR \
  --revision="fp16" \
  --with_prior_preservation --prior_loss_weight=1.0 \
  --seed=3434554 \
  --resolution=512 \
  --train_batch_size=1 \
  --train_text_encoder \
  --mixed_precision="fp16" \
  --use_8bit_adam \
  --gradient_accumulation_steps=1 \
  --learning_rate=1e-6 \
  --lr_scheduler="constant" \
  --lr_warmup_steps=0 \
  --num_class_images=50 \
  --sample_batch_size=1 \
  --max_train_steps=800 \
  --save_interval=400 \
  --save_sample_prompt="photo of smzkz" \
  --concepts_list="concepts_list.json"

thx and have a nice day
Janosch

**Comment by averad** [comment]
@janoschsimon you can use https://github.com/huggingface/diffusers/blob/main/scripts/convert_diffusers_to_original_stable_diffusion.py

I don't see your question on the Gist page you referenced for the code you are using, you might want to ask your question there as well.
